tests/test_observables.py

```diff
@@ -274,6 +275,16 @@ def test_whoisxmlapi_ip(self, mock_get=None, mock_post=None):
         ).start()
         self.assertEqual(report.get("success", False), True)
 
+    def test_inquest(self, mock_get=None, mock_post=None):
```

Comment:
2 things:
1. you don't need to duplicate (copy/paste) the same function in each different test case. Instead you can just add it under the class [CommonTestCases_observables](https://github.com/intelowlproject/IntelOwl/blob/132b573beb6779bb1f69f22b1e9690a9258d0100/tests/utils.py#L32).
2. Since we have 3 different variants of the inquest analyzer, so inside the `CommonTestCases_observables`, you should make 3 functions like `test_inquest_iocdb`, `test_inquest_repdb`, `test_inquest_dfi` and in these functions when initiating the inquest's class' object pass the required `additional_config_params` like `{"inquest_analysis": "iocdb_search"}`, `{"inquest_analysis": "rdpdb_search"}`, .. respectively.